README.md

```diff
@@ -57,6 +57,9 @@ The plugin can be added to a separate (POM-) project or your main aggregator pro
 				<dontFailOnCommit>false</dontFailOnCommit>
 				<dontFailOnUpstreamVersionResolution>false</dontFailOnUpstreamVersionResolution>
 				<upstreamDependencies>
+					<bomDependency>
```

Comment:
It is caused by maven properties creation rules. IMHO it's not possible to make it in that way:
```
<upstreamDependencies>
    <upstreamDependency>dependency1</upstreamDependency>
    <bomDependency>bom1</bomDependency>
    <bomDependency>bom2</bomDependency>
    <upstreamDependency>dependency2</upstreamDependency>
</upstreamDependencies>
```
but you can make it simpler by having two separate sections like:
```
<upstreamDependencies>
    <upstreamDependency>dependency1</upstreamDependency>
    <upstreamDependency>dependency2</upstreamDependency>
</upstreamDependencies>
<bomDependencies>
    <bomDependency>bom1</bomDependency>
    <bomDependency>bom2</bomDependency>
</bomDependencies>
```
But for me personally it was easier to have single <upstreamDependencies> section with multiple types of dependecies so I decided to have:
```

Reply:
I would go with two separate sections if that's not a big effort

Reply:
@bbad,I wrote that it is one of two possible solutions of making this kind of parameterization which is backward compatible, but I didn't choose it because it will change plugin behavior in following aspect

Reply:
I've got another proposition we can distinguish boms in another way...
Please take a look:
```

Reply:
Ok, so let's just stay with current solution